README.md

```diff
@@ -3,17 +3,17 @@
 
 ## Table of Contents
 - [Introduction](#introduction)
-- [Getting Started](#getting-started)
-- [Contributing](#contributing)
+- [End Users](#end-users)
+- [Developers](#developers)
 
 ## Introduction
 BonnyCI is a Continuous Integration service offered to developers of Open Source projects.
 
-## Getting Started
-Checkout the [getting started](getting_started) section.
+## End Users
+Check out the [end user](end_users) documentation.
 
-## Contributing
-Please see the [contribution documentation](contributing) for more information.
+## Developers
+Please see the [developer documentation](developers) for more information.
```

Comment:
Same as above. Also, let's keep the labels consistent. For instance, the word documentation is included in the label here but is not in the end user section.